### ISSUE TITLE
BMP hosted: CMSIS-DAP is not detected

### ISSUE THREAD

**Issue by JojoS62** [blacksphere/blackmagic#763]
The onboard debug probe on a LPCXpresso1769 (OM13085) is not detected by 'blackmagic -t'.
lsusb lists the device 1fc9:001d, but this cmsis-dap version reports no serial and the enumeration an detection in bmp hosted fails.

Output from USBTreeView under Win10 reports the same problem:

```
    ---------------------- Device Descriptor ----------------------
bLength                  : 0x12 (18 bytes)
bDescriptorType          : 0x01 (Device Descriptor)
bcdUSB                   : 0x110 (USB Version 1.10)
bDeviceClass             : 0x00 (defined by the interface descriptors)
bDeviceSubClass          : 0x00
bDeviceProtocol          : 0x00
bMaxPacketSize0          : 0x08 (8 bytes)
idVendor                 : 0x1FC9 (NXP Semiconductors)
idProduct                : 0x001D
bcdDevice                : 0x0100
iManufacturer            : 0x01 (String Descriptor 1)
 Language 0x0409         : "NXP"
iProduct                 : 0x02 (String Descriptor 2)
 Language 0x0409         : "NXP CMSIS-DAP"
iSerialNumber            : 0x03 (String Descriptor 3)
 *!*ERROR  String descriptor not found
bNumConfigurations       : 0x01 (1 Configuration)
Data (HexDump)           : 12 01 10 01 00 00 00 08 C9 1F 1D 00 00 01 01 02   ................
                           03 01                                             ..
```

**Comment by UweBonnes** [comment]
Please test with https://github.com/UweBonnes/blackmagic/tree/bmp_only and report back.

**Comment by UweBonnes** [comment]
Should be fixed with #807, Reopen if not.
